keyboards/acheron/shark/info.json

```diff
@@ -0,0 +1,63 @@
+{
+  "keyboard_name": "",
```

Comment:
Please fill these out :) LGTM otherwise

Reply:
Updated in commit [93280c73](https://github.com/qmk/qmk_firmware/pull/7090/commits/93280c73bdff570644d7b57400b64c7bde1970b9\)